soc/riscv/riscv-privileged/qingke/Kconfig.soc

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023 Chen Xingyu <hi@xingrz.me>
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_QINGKE
```

Comment:
Is this processor not support XIP?

Reply:
`XIP` is implied in `config RISCV` in arch/Kconfig. Do I need to explicitly select it in SoC's Kconfig?

Reply:
OK. No problem.